methods/getCacheKey.js

```diff
@@ -1,3 +1,16 @@
-module.exports = function(url) {
-  return `${this.settings.app.redis.prefix}-${url}`;
+const url = require('url');
+const qp = require('query-parse');
+
+module.exports = function(path) {
+  const tokens = path.split('?');
+  if (path.length === 1) {
+    return `${this.settings.app.redis.prefix}-${path}`;
+  }
+  const queryObject = qp.toObject(tokens[1]);
+  let querystring = '';
+  const keys = Object.keys(queryObject).sort();
+  keys.forEach((key) => {
+    querystring += `${key}:${queryObject[key]}`;
```

Comment:
This should be =. Probably means the refresh endpoint isn't tested correctly because this would never match the url passed in